apiary.apib

```diff
@@ -39,6 +39,30 @@ Final version aligned with FIWARE Release 5.2.1.
 
 # Group aiakos v1
 
+## Regions [/v1/support]
+
+Get the region list with keys
+
+### Get list [GET]
+
```

Comment:
You should describe the return/response values in order to clarify what does it mean a missign key-value (gpgkey, sshkey).

Besides, I prefer to have a more json-style approach, something like:

{
 regions: [
    {
      "regionName: "Spain2",
      "gpgkey": "2016-02-01T13:09:24.000Z",
      "sshkey": "2016-02-01T13:09:24.000Z"
    },
    {
      "regionName: "Trento2",
      "gpgkey": "2016-02-01T13:09:24.000Z"
    }
 ]
}